module.js

```diff
@@ -21,5 +22,6 @@ function updateTagColorMap(oldTag, newTag, tagColorMap) {
 }
 
 module.exports = {
-	updateTagColorMap: updateTagColorMap
+	updateTagColorMap: updateTagColorMap,
+	filter
```

Comment:
same as above, rename it to something else.

Reply:
Done